projects/igniteui-angular/src/lib/test-utils/grid-functions.spec.ts

```diff
@@ -1846,6 +1846,10 @@ export class GridFunctions {
         }
     }
 
+    public static getHeaderCellByIndex(fix: ComponentFixture<any>, index: number) {
```

Comment:
You can use the method GridFunctions.getColumnHeader , where the name of the column is needed.